Tests/SwiftParserTest/Expressions.swift

```diff
@@ -179,6 +179,13 @@ final class ExpressionTests: XCTestCase {
       \Optional.?!?!?!?.??!
       """#
     )
+
+    AssertParse(
+      #"""
+      _ = distinctUntilChanged(\ .?.status)
+      _ = distinctUntilChanged(\.?.status)
```

Comment:
Worth checking the structure here? Could even minimize to just `\.?.status` since the issue was just not parsing the key path correctly. There's a `swift-parser-cli debug-init-call` mode now.

Reply:
I dunno, we'll have diagnostics if anything goes amiss.